knative-operator/hack/update-deps.sh

```diff
@@ -13,10 +13,4 @@ go mod vendor
 # TODO: Remove this once we bump kubernetes versions.
 git apply "$ROOT_DIR/hack/manifestival.patch"
 
-rm -rf $(find vendor/ -name 'OWNERS')
-# Remove unit tests & e2e tests.
-rm -rf $(find vendor/ -path '*/pkg/*_test.go')
-rm -rf $(find vendor/ -path '*/e2e/*_test.go')
-
-# Add permission for shell scripts
-chmod +x $(find vendor -type f -name '*.sh')
\ No newline at end of file
+find vendor/ \( -name "OWNERS" -o -name "OWNERS_ALIASES" -o -name "BUILD" -o -name "BUILD.bazel" -o -name "*_test.go" \) -print0 | xargs -0 rm -f
```

Comment:
👍

Reply:
btw, is the file mode change on the scripts intended, so that they are not executable anymore  ? (see the other files in this commit, can't really comment on them directly)

Happens because you have removed the chmod line.

Reply:
For now that seems fine, yes. We can always bring that back should we actually need to execute files.